packages/components/src/RichTextEditor/RichTextEditor/RichTextEditor.module.scss

```diff
@@ -51,6 +51,10 @@
     display: inline-block;
   }
 
+  .toolbarWrapper {
+    border-bottom: inherit;
+  }
+
   /* stylelint-disable no-descending-specificity */
   .editorWrapper {
     position: relative;
```

Comment:
We should be able to clean up the styles a little more now we're inheriting the border color and get rid of all those confusing class specifiers from line 66 - 102:

```CSS
    &:hover,
    &:focus-within {
      border-color: $color-gray-600;
    }

    &.error {
      border-color: $color-red-500;

      &:hover,
      &:focus-within {
        border-color: $color-red-600;
      }
    }

    &.caution {
      border-color: $color-yellow-600;

      &:hover,
      &:focus-within {
        border-color: $color-yellow-700;
      }
    }
```